var/spack/repos/builtin/packages/chai/package.py

```diff
@@ -45,12 +46,15 @@ class Chai(CachedCMakePackage, CudaPackage, ROCmPackage):
 
     depends_on('cmake@3.8:', type='build')
     depends_on('cmake@3.9:', type='build', when="+cuda")
+    depends_on('cmake@3.14:', when='@2022.03.0:')
 
+    depends_on('blt@0.5.0:', type='build', when='@2022.03.0:')
     depends_on('blt@0.4.1:', type='build', when='@2.4.0:')
     depends_on('blt@0.4.0:', type='build', when='@2.3.0')
     depends_on('blt@0.3.6:', type='build', when='@:2.2.2')
 
     depends_on('umpire')
+    depends_on('umpire@2022.03.0', when='@2022.03.0')
```

Comment:
Could we make all of these 2022.03.0:?  With the merge back to develop on raja, and direct tag from main on camp, and that version, develop hashes should start building for us.

Reply:
Sure thing, a : for both the depends and the when?

Reply:
Yeah I think so. There still isn't a great way to specify what we really want, which is to grab the develop version when it's an unknown newer version, but with `:` on both if we manually specify a hash for both it should match rather than give us a concretization error that way.